e2e-test/src/test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,41 @@
+projectId=cdf-athena
+dataset=test_automation
+windowSize=1920x1040
+clsErrorMessageColor=#a40403
+clsPluginComment=Comment for Plugin
+clsPluginUpdateComment=CloudSQL MySQL 
+clsPluginValidateComment=Comment for Plugin
+clsPluginValidateUpdateComment=CloudSQL MySQL Comment for Plugin
+clsReferenceNameInvalid=#@#@#@#@#@
+clsReferenceNameValid=TestReference
+clsDriverNameInvalid=@@!67RTC
+clsDriverNameValid=cloudsql-mysql
+clsDatabaseName=cdfdb
+cloudSqlInstance=cdf-athena:europe-west1:cdf-qa-testdb
+ipTypes=PRIVATE
+clsConnectionNameInvalid=10
+clsImportQuery=select * from cdftest4;
+clsInvalidImportQuery=import query;
+clsImportQuery1=select * from cdftest4 where $CONDITIONS;
+clsImportQuery2=select * from  cdftest4 where parent_id=2056;
+clsBoundingQuery=select MIN(s_no),MAX(s_no) from cdftest4;
+clsConnectionNameValid=10.232.15.232
+clsConnectionTimeout=20
+clsSplitColumn=s_no
+clsIncorrectSplit=@#*()
+clsNumberOfSplits=3
+clsInvalidNumberOfSplits=0
+clsTableNameBQCS=cdftest2
+clsTableNameGCSCS=cdftest3
+clsTableNameBQCS1=cdftest4
+clsPathGCS=gs://cdf-athena-test/cloudmysql
+clsDataset=test_abc
+clsProjectID=cdf-athena
+clsTableNameBQ=tableabc
+clsTableNameBQ1=tablecloud
+clsBucket=gs://cdf-athena-test/TestingGCS.csv
```

Comment:
load data from the repo instead of depending on data existing in a GCS bucket. For example, see google-cloud tests - https://github.com/data-integrations/google-cloud/tree/develop/src/e2e-test/resources/testdata